ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -294,6 +294,17 @@ objects:
             --junit-dir /tmp/artifacts/junit -f -
         }
 
+        function run-csi-tests() {
+          for url in $(curl -s https://api.github.com/repos/openshift/ovirt-csi-driver-operator/contents/manifests | jq '.[].download_url'); do
+            echo ${url}| xargs curl -L| oc create -f -
+            sleep 30s
+          done
+          oc apply -f /tmp/cluster/storageclass.yaml
```

Comment:
We get an error about the storageclass apply, because the operator already creates one with that name. 
What we should do is to change the name in cluster/storageclass.yaml and make sure our storage claim for the test uses that in `spec.storageClassName`

Reply:
btw just to be safe we need to set the initial one to non-default
` oc patch storageclass ovirt-csi-sc -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'`

Reply:
the storage class section is not needed anymore - we are setting one out of the box.